CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## main / unreleased
 
+* [CHANGE] Store-gateway: skip verifying index header integrity upon loading. To enable verification set `blocks_storage.bucket_store.index_header.verify_on_load: true`.
```

Comment:
```suggestion
### Grafana Mimir

* [CHANGE] Store-gateway: skip verifying index header integrity upon loading. To enable verification set `-blocks_storage.bucket_store.index_header.verify_on_load=true`.
```

Reply:
I copied all the sections from the previous release and added them to the unreleased section in a69eed43afe5ae44378259d514f65136f2d93203